plone/namedfile/utils/__init__.py

```diff
@@ -269,24 +268,21 @@ def rotate_image(image_data, method=None, REQUEST=None):
 
 
 def getRetinaScales():
-    registry = queryUtility(IRegistry)
-    if not registry:
-        return []
-    settings = registry.forInterface(
-        IImagingSchema, prefix='plone', check=False)
-    if settings.retina_scales == '2x':
-        return [{
-            'scale': 2,
-            'quality': settings.quality_2x,
-        }]
-    if settings.retina_scales == '3x':
-        return [
-            {
-                'scale': 2,
-                'quality': settings.quality_2x,
-            },
-            {
-                'scale': 3,
-                'quality': settings.quality_3x,
-            }]
+    try:
+        from Products.CMFPlone.interfaces.controlpanel import IImagingSchema
```

Comment:
I prefer to have that import on module level, not nested in this function.
I often do:
```
try:
    from Products.CMFPlone.interfaces.controlpanel import IImagingSchema
except ImportError:
    IImagingSchema = None
...
if IImagingSchema is not None:
    ....
```